### ISSUE TITLE
page disappears after using webContents.printToPDF with overflow:hidden style

### ISSUE THREAD

**Issue by Eusebius1920** [electron/electron#4119]
If you have the following css:

```
html, body{
 margin: 0;
 padding: 0;
 height: 100vh;
 overflow: hidden;
}
```

where `overflow: hidden;` disables the "elastic scroll", electron crashes if you call webContents.printToPDF with the default options.

Seen on MacOSX Yosemite 10.10.5.

**Comment by zcbenz** [comment]
What's the version of Electron? Can you paste the crash report?

**Comment by Eusebius1920** [comment]
I tried all versions since 0.35.4
Electron does not close, but the Browser-Window goes grey and sometimes the Devtool disconnect.

How can I get a crash report to post it?
The last thing in Devtools written is "Fail to generate PDF".

**Comment by zcbenz** [comment]
You can find the crash report in the `Console.app`.

I'm not able to reproduce by adding the css rule to the default app, it is probably the other parts of your code caused the problem, can you also provide an example?

**Comment by Eusebius1920** [comment]
Replacing the default_app/index.html with this one (and not touching anything else) on 0.36.4:
```
<html>
<head>
  <title>Electron</title>
  <style>
    html, body{
		margin: 0;
		padding: 0;
		height: 100vh;
		overflow: hidden;
	}
  </style>
  <script>
	function generatePDF(){
		var electron = require('electron');
		var wind = electron.remote.getCurrentWindow()
		wind.webContents.printToPDF({}, function(err, data){console.log(err);});
	}
  </script>
</head>
<body>
<button onclick="generatePDF();">Generate PDF</button> 
 This is some minimal "not-working" example app... Just click the button. It also happens if fired from main.js. <br>
 If you run generatePDF() twice, the Devtools will disconnect from target.
</body>
</html>
```
makes electron "crash" for me. I tested this on Windows 7 with 0.36.4. (As already written, the whole failure, but not this particular example, is also seen on MacOSX Yosemite with all version 0.35.4 until 0.36.4.

**Comment by zcbenz** [comment]
I can reproduce with the page provided by you, it is interesting that the window is not crashed, it is just the whole page disappeared.

**Comment by Eusebius1920** [comment]
If you fire the code via DevTools twice, i.e.:
```
generatePDF();
generatePDF();
```
the "crash" gets worse.

**Comment by MrSaints** [comment]
I'm getting a similar error ("Fail to generate PDF") sometimes when running Electron in headless through Xvfb. It rarely happens so I wasn't able to catch it.

**Comment by deepak1556** [comment]
the problem seems to be with viewport units and printing, related upstream issue https://code.google.com/p/chromium/issues/detail?id=382313.

**Comment by codebytere** [comment]
Fixed as of `53.0.2785.116` (we use a version more recent than this in every supported Electron major version) per https://code.google.com/p/chromium/issues/detail?id=382313.
